datadog/data_source_datadog_organization_settings.go

```diff
@@ -0,0 +1,160 @@
+package datadog
+
+import (
+	"context"
+
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceDatadogOrganizationSettings() *schema.Resource {
+	return &schema.Resource{
+		Description: "Use this data source to retrieve information about an existing organization.",
+		ReadContext: dataSourceDatadogOrganizationRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "Name for Organization.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"public_id": {
+				Description: "The `public_id` of the organization you are operating within.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"description": {
+				Description: "Description of the organization.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"settings": {
```

Comment:
What's the use case for an organisation settings datasource ?

Reply:
It allows reading the `settings` of the organization. It will be up to the provider's user to decide what to do with those values.

Reply:
Right, I was asking if you had a specific use case in mind, if this datasource would be useful at all.

Reply:
Ye. So I don't have any use case here. But maybe someone will find it useful. I've just exposed all values which are returned by the API

Reply:
Exactly, I'm failing to see a use for it too, I say let's remove the datasource until a user actually has a use for it.

Reply:
Personally, writing Terraform provider is like writing SDK. You are just providing a programmatic way to access resources. It is up to the users to find a use case. But I'll remove it in next change.